Library/Homebrew/utils/curl.rb

```diff
@@ -61,7 +61,7 @@ def curl_download(*args, to: nil, continue_at: "-", **options)
 end
 
 def curl_output(*args, **options)
-  Open3.capture3(*curl_args(*args, show_output: true, **options))
+  Open3.capture3(curl_executable, *curl_args(*args, show_output: true, **options))
```

Comment:
Might be nice if (eventually) capturing output like this is something possible with `SystemCommand`

Reply:
Done.

Reply:
@reitermarkus Nice! I think it would be cool if eventually all the calls to `system`, `Formula#system`, `Utils.popen*` end up wrapping `system_command` calls.